cmd/harmony/main.go

```diff
@@ -273,6 +273,9 @@ func setupStakingNodeAccount() error {
 	return nil
 }
 
+// MaxNumBLSKeysPerNode specifies the maximum number of bls keys that a node can run
+const MaxNumBLSKeysPerNode = 4
```

Comment:
don't hardcode the number here.  This configuration variable could be network specific, we may use different number for testnet/devnet, etc.  So, I think you should add it as a network configuration parameter.

Reply:
you mean in `internal/configs/sharding` or `internal/params/config.go` ?

Reply:
@LeoHChen ?

Reply:
as in a main.go flag.